File: frontend/src/app/njegovatelj/getAllNjegovateljiResponse.ts
import {GetAllKorisnickiNalogResponseKorisnickiNalog} from "../korisnicki-nalog/getAllKorisnickiNalogResponse";

export interface GetAllNjegovateljiResponse {
    njegovatelji: GetAllNjegovateljaResponseNjegovatelj[];
}
export interface GetAllNjegovateljaResponseNjegovatelj {
    zaposlenikId:number
    imePrezime: string;
    jmbg: string;
    datumRodjenja: Date;
    datumZaposlenja: Date;
    nalogId: number|null;
    poslovnaPozicijaId: number;
    brojPacijenata: number;
    isMedicinskiTehnicar: boolean;
    isNjegovatelj: boolean;
}

File: DomZaStaraLicaApi/Endpoints/Njegovatelj/GetAllNjegovatelji/GetAllNjegovateljiResponse.cs
namespace DomZaStaraLicaApi.Endpoints.Njegovatelj.GetAllNjegovatelji
{
    public class GetAllNjegovateljiResponse
    {
        public List<GetAllNjegovateljiResponseNjegovatelj> Njegovatelji { get; set; }
    }
    public class GetAllNjegovateljiResponseNjegovatelj
    {
        public int ZaposlenikId { get; set; }
        public string ImePrezime { get; set; }
        public string JMBG { get; set; }
        public DateTime DatumRodjenja { get; set; }
        public DateTime DatumZaposlenja { get; set; }
        public int? NalogId { get; set; }
        public int PoslovnaPozicijaId { get; set; }
        public int brojPacijenata { get; set; }
        public bool isMedicinskiTehnicar { get; set; }
        public bool isNjegovatelj { get; set; }
    }
}
